<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrbitLiveFunctions</class>
 <widget class="QWidget" name="OrbitLiveFunctions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>284</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="FilterLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="OrbitTreeView" name="treeView">
     <attribute name="headerDefaultSectionSize">
      <number>0</number>
     </attribute>
     <attribute name="headerMinimumSectionSize">
      <number>0</number>
     </attribute>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="iteratorLayout">
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OrbitTreeView</class>
   <extends>QTreeView</extends>
   <header>orbittreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
